mpi/Makefile.am

```
@@ -29,8 +29,13 @@ MPI_NAMESPACES = mpi
 
 MPI_EXTRACTED = $(MPI_NAMESPACES:=_api.yaml)
 
+if ENABLE_CLANG_PARSER
+$(MPI_EXTRACTED): $(MODIFIED_OMP_HDR) $(srcdir)/mpi.h.include
```

This dependency is wrong
```suggestion
$(MPI_EXTRACTED): $(MODIFIED_MPI_HDR) $(srcdir)/mpi.h.include
```